NVIDIA-918: Add NVIDIA DPF Operator documentation (#117142)
openshift/openshift-docs: modules/nw-dpf-installing-dpf-operator.adoc

// Module included in the following assemblies:
//
// * networking/networking_operators/dpf_operator/dpf-operator-installation.adoc

:_mod-docs-content-type: PROCEDURE
[id="nw-dpf-installing-dpf-operator_{context}"]
= Install the DPF Operator

[role="_abstract"]
You can install the DPF Operator by using Helm to deploy the operator into the `dpf-operator-system` namespace on your management cluster.

[IMPORTANT]
====

> You must install the DPF Operator before the `dpf-hcp-provisioner-operator` because the provisioner operator requires the following DPF custom resource definitions to be available: `DPUCluster`, `DPUFlavor`, `DPUDeployment`, and `DPFOperatorConfig`.

Reviewer: Definitely think this belongs here as well as the earlier module.

====

.Prerequisites

* You have access to the management cluster as a user with the `cluster-admin` role.

> * The OpenShift CLI (`oc`) is installed.

Reviewer: ```suggestion
* You have installed the `oc` CLI.
```

> * The Helm CLI (`helm`) is installed.

Reviewer: ```suggestion
* You have installed the `helm` CLI.
```

> * The DPF Operator environment variables are set. For details, see "DPF Operator installation environment variables".

Reviewer: ```suggestion
* You have set the DPF Operator environment variables. For details, see "DPF Operator installation environment variables".
```
